request/builder.js

```diff
@@ -72,8 +72,14 @@ class CertificateRegistrationRequestBuilder {
     request.setCertHolderId(this.certHolderId);
     request.setCertVersion(this.certVersion);
     request.setCertPem(this.certPem);
+
     return request;
   }
+
+  async getByteArray() {
+    const request = await this.build();
+    return request.serializeBinary();
```

Comment:
BTW, I think it's weird to call `serializeBinary` in a Builder class.
Builder's responsibility should be to just create a request and `ClientServiceBase` should call `serializeBinary` with the request. Thought ?

Reply:
Yeap, it would be more proper for builder to just build the request.
As suggested by Plenty, we can implement
```
/**
   * @return {Promise<void>}
   */
  async buildCertificateRegistrationRequest() {
    const builder = new CertificateRegistrationRequestBuilder(
        new this.protobuf.CertificateRegistrationRequest(),
    ).withCertHolderId(this.certHolderId)
        .withCertVersion(this.certVersion)
        .withCertPem(this.certPem);

    return await builder.build();
  }
```
at the ClientServiceBase and for all the other request as well...

Reply:
typo ?
```this.buildCertificateRegistrationRequest()```

Reply:
oh, that's the correct method name...

Reply:
Regarding the method name,
how about this ?
```
async createSerializedCertificateRegistrationRequest() {
}
```

Reply:
Hmm sounds great to me, as it also indicates what is inside the returned object...